rskj-core/src/test/java/co/rsk/peg/PegTestUtils.java

```diff
@@ -272,6 +273,19 @@ public static Federation createFederation(BridgeConstants bridgeConstants, List<
         );
     }
 
+    public static Federation createP2shErpFederation(BridgeConstants bridgeConstants, ActivationConfig.ForBlock activations, List<BtcECKey> federationKeys) {
+        federationKeys.sort(BtcECKey.PUBKEY_COMPARATOR);
+        return new P2shErpFederation(
+            FederationTestUtils.getFederationMembersWithBtcKeys(federationKeys),
+            Instant.ofEpochMilli(1000L),
+            0L,
+            bridgeConstants.getBtcParams(),
+            bridgeConstants.getErpFedPubKeysList(),
+            bridgeConstants.getErpFedActivationDelay(),
+            activations
```

Comment:
Could we pass a mock for activations object here and avoid receiving it as param? It doesn't change anything for P2SH ERP federations

Reply:
Done